### PR TITLE
Added meetup api member search and sync to update user data with meetup data

### LINKED ISSUE #87: Create job to merge Meetup user data into Postgres
# Feature
## Why is this feature being added?
@rickr, we have a lot of data in Meetup that should be synced up with our regular user database, regarding members that go to our regular meetups, chapters and rate of growth, etc.

## What should your feature do?

### PR DESCRIPTION
**Pull request by BRIMIL01**
# Description of changes
Added meetup api member search and sync to update user data with meetup data. I also added an editorconfig so my editor would "do the right thing" in regards to the rubocop settings. I wasn't able to actually make a call against the meetup api so my fixture data is best guess.

# Issue Resolved
Fixes #87

### CODE REVIEW

File: test/support/meetups/sample_api_responses.rb [hunk 1]
```diff
@@ -36,32 +42,32 @@ def event_endpoint_parsed_response
...
-        'lat' => 36.844764709472656,
-        'lon' => -75.97899627685547,
-        'repinned' => false,
-        'address_1' => '1701 Baltic Avenue',
-        'city' => 'Virginia Beach',
-        'country' => 'us',
-        'localized_country_name' => 'USA',
-        'zip' => '',
-        'state' => 'VA'},
+        {'id' => 24628300,
```
Comment (BRIMIL01) [inline test/support/meetups/sample_api_responses.rb]: My editor started a whitespace war :-(

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# EditorConfig is awesome: https://EditorConfig.org
```
Comment (robbkidd) [inline .editorconfig]: not gonna be part of your system!

(👍)

File: app/jobs/meetup_member_sync.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+class MeetupMemberSync
```
Comment (robbkidd) [inline app/jobs/meetup_member_sync.rb]: This job is sensible. I suggest a tweak to the naming, for it to end in `Job` to match the current naming pattern for the backend classes that are background jobs.

I'll offer `LocateUsersWithDataFromMeetupJob` unless there are thoughts about this worker being responsible other types of user data in the future.

Comment (BRIMIL01) [reply app/jobs/meetup_member_sync.rb]: Possibly `MeetupMemberToUserSyncJob`?

Comment (robbkidd) [reply app/jobs/meetup_member_sync.rb]: Maybe? That reads to me like we're creating users or something. `MeetupMemberSyncJob` could be fine if we want to keep the details of the sync out of the name.

Comment (wimo7083) [reply app/jobs/meetup_member_sync.rb]: `LoCaTe_MemBeRsMEeTUp4MeMBErs`

Comment (robbkidd) [reply app/jobs/meetup_member_sync.rb]: :shipit:

Comment (BRIMIL01) [reply app/jobs/meetup_member_sync.rb]: Names are hard

Comment (wimo7083) [reply app/jobs/meetup_member_sync.rb]: So this updates members if there is new meetup data?

`UpdateMembersMeetupDataJob`?

Comment (BRIMIL01) [reply app/jobs/meetup_member_sync.rb]: It doesn't overwrite, it just adds any missing data from a meetup call

Comment (robbkidd) [reply app/jobs/meetup_member_sync.rb]: I would describe the current behavior as "update a user with information about them from Meetup if they are a member of a Meetup." That information is currently location info nothing specific about Meetup, but could become more later.

Comment (BRIMIL01) [reply app/jobs/meetup_member_sync.rb]: Sure, maybe `ExternalUserAccountUpdateJob`

Comment (robbkidd) [reply app/jobs/meetup_member_sync.rb]: My vote is to add Job to the current class name and revisit the name when we want it to do more.

Comment (BRIMIL01) [reply app/jobs/meetup_member_sync.rb]: Done

**Pull request merged**